File: app/src/main/java/com/spidroid/starry/ui/messages/CommunitiesFragment.java
package com.spidroid.starry.ui.messages;

import android.view.LayoutInflater;
import android.view.ViewGroup;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import androidx.fragment.app.Fragment;
import com.spidroid.starry.R;

public class CommunitiesFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_placeholder, container, false);
        TextView textView = view.findViewById(R.id.tv_placeholder);
        textView.setText("Communities coming soon! 🚀");
        return view;
    }
}

File: app/src/main/java/com/spidroid/starry/ui/messages/CommunitiesFragment.kt
package com.spidroid.starry.ui.messages

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import com.spidroid.starry.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class CommunitiesFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        val view = inflater.inflate(R.layout.fragment_placeholder, container, false)
        val textView = view.findViewById<TextView>(R.id.tv_placeholder)
        textView.setText("Communities coming soon! 🚀")
        return view
    }
}